packages/quantic/force-app/main/default/lwc/quanticColoredResultBadge/quanticColoredResultBadge.js

```diff
@@ -0,0 +1,113 @@
+import {LightningElement, api} from 'lwc';
+import {RGBToHSL, HEXToRGB, invalidRGBValues} from './colorsUtils';
+
+const defaultPrimaryColor = '#FFF7BA';
+const defaultSecondaryColor = '#E2B104';
+const lightnessDegree = 48;
+
+/** @typedef {import("coveo").Result} Result */
+
+/**
+ * The `QuanticColoredResultBadge` component displays a colored badge showing a label.
+ * If the `Label` property is set in this component, its value will be displayed in the badge, Otherwise the `Result` and 'Fieldname' properties are required and the value of the given field will lbe displayed.
```

Comment:
```suggestion
 * If the `Label` property is set in this component, its value will be displayed in the badge, Otherwise the `Result` and `Fieldname` properties are required and the value of the given field will lbe displayed.
```